ebean-core/src/main/java/io/ebeaninternal/server/executor/DefaultBackgroundExecutor.java

```diff
@@ -106,23 +106,23 @@ public void execute(Runnable task) {
 
   @Override
   public ScheduledFuture<?> scheduleWithFixedDelay(Runnable task, long initialDelay, long delay, TimeUnit unit) {
-    return schedulePool.scheduleWithFixedDelay(wrap(logExceptions(task)), initialDelay, delay, unit);
+    return schedulePool.scheduleWithFixedDelay(logExceptions(task), initialDelay, delay, unit);
   }
 
   @Override
   public ScheduledFuture<?> scheduleAtFixedRate(Runnable task, long initialDelay, long delay, TimeUnit unit) {
-    return schedulePool.scheduleAtFixedRate(wrap(logExceptions(task)), initialDelay, delay, unit);
+    return schedulePool.scheduleAtFixedRate(logExceptions(task), initialDelay, delay, unit);
   }
 
   @Override
   public ScheduledFuture<?> schedule(Runnable task, long delay, TimeUnit unit) {
-    return schedulePool.schedule(wrap(logExceptions(task)), delay, unit);
```

Comment:
I'm not sure, if we should keep it here. The task will be scheduled one time. But this may keep objects also long in memory.
Maybe the caller is responsible to pass a proper runnable/callable

Reply:
I'm not sure either - it almost looks like a 50/50 call to me. 

If we do wrap() here it would be a problem if the delay is _"a long time"_, and if that is a problem for a particular use case then what is the workaround? I don't see a nice workaround so that is suggesting to me that we don't wrap() here.